README.md

```diff
@@ -8,6 +8,12 @@ In-App Messaging (IAM) module allows app developers to easily configure and disp
 
 This module supports iOS 12.0 and above. It has been tested with iOS 12.5 and above.
 
+# Requirements
+
+Xcode 12.5.x or Xcode 13+
```

Comment:
13+ means 14 as well, isn't it? Maybe it's better to say Xcode 13.x ?

Reply:
Yes, that's deliberate. We should support the next Xcode version when it is officially released.